lib/hanami/utils/hash.rb

```diff
@@ -121,7 +121,7 @@ def self.deep_stringify(input) # rubocop:disable Metrics/MethodLength
         end
       end
 
-      # Deep duplicate hash values
+      # Shallow duplicates hash values
```

Comment:
Honestly, not sure about that one. description and naming seem a bit contradicting

Reply:
> description and naming seem a bit contradicting

The semantic of this method is: duplicate (as in `#dup`) all the nested values of the input hash.

Trivial example:

```ruby
hash = { a: [1, 2, 3] }
hash.fetch(:a).object_id
# => 70340898593420
duplicated = Hanami::Utils::Hash.deep_dup(hash)
# => {:a=>[1, 2, 3]}
duplicated.fetch(:a).object_id
# => 70340898530980
```

---

What's the best way to express it?

Reply:
So it behaves like classic deep_dup. Then I think it would make sense to reflect that in the name as well as in description, which now says: 

> The output of this function is a shallow duplicate of the input.

will push such change.